django_components/component.py

```diff
@@ -60,9 +62,93 @@ def __new__(mcs, name, bases, attrs):
             if hasattr(media, "js") and isinstance(media.js, str):
                 media.js = [media.js]
 
+        _resolve_component_relative_files(attrs)
+
         return super().__new__(mcs, name, bases, attrs)
 
 
+def _resolve_component_relative_files(attrs: dict):
```

Comment:
I've defined the logic as standalone functions and prefixed with underscore. Let me know if there are any styling guidelines I should follow

Reply:
Makes sense that this is a private function.